README.md

```diff
@@ -49,6 +47,15 @@ The scripts will:
    that are required for the OpenShift installer - for example,
```

Comment:
"for the OpenShift installer - for example" - Change to "for the OpenShift installer. For example,